client_handler.cpp

```diff
@@ -93,14 +93,21 @@ void ClientHandler::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 	PROC_TIME(OnAfterCreated)
 
 #if CHROME_VERSION_MAJOR >= 126
+	//Chrome runtime styleのとき、Jitsiのミーティングルームにおいて、Googleアカウント経由でログインする際、「パスワードを保存」
+	//プロンプトが表示されるとChronosがクラッシュする。パスワードマネージャー機能を無効化し、プロンプトが表示されないようにする。
+	CefRefPtr<CefRequestContext> requestContext = browser->GetHost()->GetRequestContext();
+	CefString error;
+	CefRefPtr<CefValue> value = CefValue::Create();
+	value->SetBool(false);
+	requestContext->SetPreference("credentials_enable_service", value, error);
+	value->SetBool(false);

```

Comment:
This assignment seems redundant.

Reply:
NOTE: popup window was kept when it succeeded.

Reply:
> This second assignment (`value->SetBool(false)`) seems redundant.

It was intended.
I thought it was better to explicitly set values for each `SetPreference`.

How do you think about that?

Reply:
> NOTE: popup window was kept when it succeeded.

It is reproduced on my environment.
I am investigating it now...

Reply:
> How do you think about that?

If value is not kept before/after SetPreference, it is reasonable to set explicitly.
Okay.

Reply:
> NOTE: popup window was kept when it succeeded.

Let's resolve in another PR.